app/scripts/views/sign_up.js

```diff
@@ -122,7 +127,8 @@ function (Cocktail, _, p, BaseView, FormView, Template, AuthErrors, mailcheck,
     },
 
     events: {
-      'blur input.email': 'suggestEmail'
+      'blur input.email': 'suggestEmail',
+      'blur input.password': '_checkPasswordStrength'
```

Comment:
I assume this should be the A/B path here, could this be 'blur input.password': 'blurPassword' or 'passwordBlur'?

Inside `blurPassword` (or whatever other name) we do other things, where one of them is the A/B check for enrolment into password strength experiment